MIGRATION_ANALYSIS.md

```
@@ -0,0 +1,810 @@
+#### Trigger Actions (5)
+1. **MOUNT** component
+2. **CLICK** #populate-notifications
+3. **CLICK** #drawer-toggle
+4. **CLICK** [aria-label="Notification actions dropdown"] (first)
+5. **CLICK** [role="menuitem"] containing "Mark as read" (first)
+
+#### Assertions (2)
+- Before action: `.pf-m-read` should have length 0
+- After action: `.pf-m-read` should have length 1
+
+### Migration Recommendation
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)